modules/cli/pages/cli-intro.adoc

```diff
@@ -39,46 +39,117 @@ You can collect diagnostics through the command-line interface by using the xref
 [#server-tools-packages]
 == Server Tools Packages
 
-For convenience, Couchbase provides a  tools package  that contains the following utilities:
+For convenience, Couchbase provides Server developer tools package and Server admin tools package.
+
+=== Server Developer Tools Packages
+
+Server developer tools package contains the following utilities:
```

Comment:
I like how the current doc notes that these are EE Server utilities.
So, something like:
Server developer tools package contains the following EE Server utilities: